core/src/nucleus/mod.rs

```diff
@@ -92,35 +93,6 @@ impl EntrySubmission {
     }
 }
 
-/// Dispatch ExecuteZoneFunction to and block until call has finished.
```

Comment:
I did some searches for this and I guess it wasn't even being called anywhere? If so, good you removed it.